svgnative/include/SVGRenderer.h

```diff
@@ -218,6 +218,7 @@ class Path
 
     virtual void Rect(float x, float y, float width, float height) = 0;
     virtual void RoundedRect(float x, float y, float width, float height, float cornerRadius) = 0;
+    virtual void RoundedRect(float x, float y, float width, float height, float cornerRadiusX, float cornerRadiusY) = 0;
```

Comment:
If we are changing (by adding new functions to) the SVGRenderer, why not just have one RoundedRect function. Deriving classes can optimize/differ between rx == ry and rx != ry.

Reply:
Also, please add tests to make sure we execute all the cases of (non-)rounded rects.

Reply:
So, should I replace existing RoundedRect(x, y, w, h, r) by RoundedRect(x, y, w, h, rx, ry)?

Reply:
yes. I think so.